coordinator/internal/logic/submitproof/proof_receiver.go

```diff
@@ -133,6 +137,18 @@ func NewSubmitProofReceiverLogic(cfg *config.ProverManager, chainCfg *params.Cha
 			Name: "coordinator_validate_failure_submit_have_been_verifier",
 			Help: "Total number of submit proof validate failure proof have been verifier.",
 		}),
+		evmCyclePerGas: promauto.With(reg).NewGauge(prometheus.GaugeOpts{
```

Comment:
Suggest you also add 3 columns to prover_task table. sometimes we need to know the specific task which affect the whole metric

Reply:
These new metrics come from https://github.com/scroll-tech/scroll/pull/1726

Reply:
Maybe we can handle it later. Has added the [feat request issue](https://github.com/scroll-tech/scroll/issues/1728)